project/Project.scala

```diff
@@ -76,11 +76,7 @@ object Zipkin extends Build {
     crossPaths := false,            /* Removes Scala version from artifact name */
     fork := true, // forking prevents runaway thread pollution of sbt
     baseDirectory in run := file(cwd), // necessary for forking
-    resolvers := Seq(
-      Resolver.jcenterRepo,
-      Resolver.typesafeRepo("releases"),
-      "Twitter Maven Repository" at "https://maven.twttr.com/" // for thrift 0.5
-    ),
+    resolvers += "twitter-repo" at "http://maven.twttr.com", // for thrift 0.5
```

Comment:
Scratch that... Something is falling through to the Twitter repo.. I will
look into this!

```

[error] SERVER ERROR: Service Temporarily Unavailable url=
http://maven.twttr.com/org/slf4j/slf4j-parent/1.6.4/slf4j-parent-1.6.4.jar

```